tests/zeroconfig/windows/testdata/resource_traces/aspnetcore.yaml

```diff
@@ -21,3 +21,4 @@ resource_spans:
         net.peer.ip: <ANY>
         signalfx.tracing.library: dotnet-tracing
         signalfx.tracing.version: <ANY>
+        splunk.zc.method: <ANY>
```

Comment:
fwiw this should support something like

```suggestion
        splunk.zc.method: <RE2(signalfx-dotnet-tracing-.*)>
```

Reply:
Thanks! Was wondering if it supported that.

Reply:
Sorry, I didn't add the default value loader to the ResourceTraces so this is untrue atm.